tracpro/orgs_ext/tasks.py

```diff
@@ -229,3 +230,57 @@ def wrap_logger(self, level, org, msg, *args, **kwargs):
         kwargs.setdefault('org', org.name)
         msg = "{task} for {org}: " + msg
         return super(OrgTask, self).wrap_logger(level, msg, *args, **kwargs)
+
+
+@task(ignore_result=True)
+def fetch_runs(org_id, since):
+    """
+    Fetch responses for the org with id=org_id, going back
+    to `since` (datetime).
+
+    Creates or updates Response objects for each run.
+    """
+    from tracpro.polls.models import Poll, Response  # Avoid circular imports
+
+    logger.debug("Running fetch_runs")
+
+    try:
+        org = Org.objects.get(id=org_id)
+    except Org.DoesNotExist:
+        raise ValueError("No such org with id %d" % org_id)
+    #
+    # if not any([days, hours, minutes]):
+    #     raise ValueError("At least one of days, hours, or minutes must be non-zero")
+    #
+    # since = timezone.now() - relativedelta(minutes=minutes, hours=hours, days=days)
+
```

Comment:
Good grief. I review my code so carefully for things like this commented out code before opening a pull request, and still they slip through. Will delete.